Marlin/src/lcd/menu/menu_info.cpp

```diff
@@ -49,26 +49,29 @@
     );
 
     char buffer[21];
+
     printStatistics stats = print_job_timer.getStats();
 
+    #define STATIC_PAIR(MSG, VALUE, CNTR)    do {strcpy_P(buffer, PSTR(": ")); strcpy(buffer, VALUE); STATIC_ITEM(MSG, CNTR, false, buffer);} while(0)
+    #define STATIC_PAIR_P(MSG, PVALUE, CNTR) do {strcpy_P(buffer, PSTR(": ")); strcpy_P(buffer, PSTR(PVALUE)); STATIC_ITEM(MSG, CNTR, false, buffer);} while(0)
```

Comment:
It looks like the second `strcpy` in each of these should be `strcpy(buffer+2 ...)`.